src/common/com/intellij/plugins/haxe/HaxeBundle.properties

```diff
@@ -171,14 +170,80 @@ haxe.semantic.method.parameter.optional.add=Declare parameter as optional
 haxe.semantic.method.parameter.optional.remove=Remove "optional" mark from parameter declaration
 haxe.semantic.implemented.super.method.signature.differs=Signature of inherited method ''{0}'' declared by class ''{1}'' differs from declared by interface. Expected ''{2}'', got ''{3}''
 haxe.semantic.static.field.override=Field ''{0}'' overrides a static field of a superclass.
-haxe.semantic.variable.redefinition=Redefinition of variable ''{0}'' in subclass is not allowed. Previously declared at ''{1}''.
-haxe.semantic.property.cant.be.final=Property can not be final
+haxe.semantic.no.constructor=Class {0} doesn't have a constructor
+haxe.semantic.incompatible.types=\
+  <html><body>Incompatible types.<table>\
+  <tr><td>Expected:</td><td><b>{1}</b></td></tr>\
+  <tr><td>Found:</td><td>{0}</td></tr>\
+  </table></body></html>
+haxe.semantic.more.arguments.than.expected=Found more arguments than expected
+haxe.semantic.0.arguments.expected.got.1=Expected {0} arguments count, got {1}
+haxe.semantic.condition.must.be.boolean=Condition must be boolean type
+semantic.static.constructor=Constructor can't be static
+haxe.semantic.not.anonymous.type=Not an anonymous type
+haxe.semantic.cant.extend.self=Cannot extend self
+haxe.semantic.variable.redefinition.not.allowed=Redefinition of variable ''{0}'' in subclass is not allowed. Previously declared at ''{1}''
+haxe.semantic.package.not.correspond.file.path=Package name ''{0}'' does not correspond to the file path ''{1}''
+haxe.semantic.package.name.uppercase=Package name ''{0}'' must start with a lower case character
+haxe.semantic.static.initializer.must.be.static=Static initializer method ''{0}'' must be static
+haxe.semantic.method.body.not.allowed=Method body not allowed for methods declared in {0}
+haxe.semantic.return.type.required=Return type required for method declared in {0}
+haxe.semantic.parameter.type.required=Parameter type required for method declared in {0}
+haxe.semantic.type.name.uppercase=Type name ''{0}'' must start with uppercase
+haxe.semantic.duplicate.member.name=Duplicate class member declaration ''{0}''
+haxe.semantic.not.allowed.override=Can't override static, inline or final methods
+haxe.semantic.overrides.nothing=Method overrides nothing
+haxe.semantic.duplicate.parameter.name=Duplicate parameter name ''{0}''
+haxe.semantic.invalid.accessor.type=Invalid {0}ter accessor type. Only ''{0}'', ''default'', ''null'', ''never'', ''dynamic'' types allowed
+haxe.semantic.accessor.method.not.found=Accessor method ''{0}'' not found
+haxe.semantic.not.real.var=This field cannot be initialized because it is not a real variable
+haxe.semantic.optional.mark.redundant=Optional mark is redundant when initial value specified
+haxe.semantic.method.visibility.less.then.parent=Method ''{0}'' has less visibility ({1}) than declared at ''{2}'' (was <b>{3}</b>)
+haxe.semantic.not.implemented.members=<html><body>Following members, declared at interface ''{0}'' not implemented:{1}</body></html>
```

Comment:
... `The following members, declared at interface ''{0}'' are not implemented: {1}` ...